### ISSUE TITLE
Story arcs can contain the list of series, volume and issues along with their reading order.

### ISSUE THREAD

**Issue by mcpierce** [comixed/comixed#44]
As a user, when I am looking at a series / arc (especially larger cross-overs such as Maximum Carnage, or World War Hulks), I would like to be able to identify the order in which to read comics.

Likely, this would need to be a user-defined value - as I have been unable to find reading orders (or similar) in the ComicVine API lists.

This may prove to be slightly larger than anticipated...as it's entirely likely that other users may wish to group comics their own way, with their own reading orders.

## Comment from original issue

So it's surprisingly easy to do (but harder to figure out) how to get the issues for a story arc. But it's likely going to need some thought to implement.

Basically, you have to find the ID for the story arc using the story_arcs REST API, then with it's ID you can query that specific story arc. The response returns a list of issues within that story arc. 

For the "Kree/Shiar War" story arc, the query:

```https://comicvine.gamespot.com/api/story_arc/4045-27758/?api_key=[API-KEY]&format=json```

returns the list of issues. But each issue entry only has the CV ID for the issue and not the details.

So my first thought is, we could just query and store the story arcs in CX'd database. Then  when getting comics we would have to then check if the related story arc is in that table, etc. Not impossible but definitely needs some thought.
